js/app.js

```diff
@@ -851,9 +851,6 @@ cacaoApp.controller('GAFDetailCtrl', ['$scope', '$routeParams', 'CacaoBackend',
 
             $scope.gaf.db_reference = parseInt($scope.gaf.db_reference.replace('PMID:', ''));
             $scope.gaf.show_qualifier = $filter('qualifier_to_text')($scope.gaf.qualifier);
-            if (!$scope.gaf.notes) {
-                 $scope.gaf.notes = "None";
-            }
```

Comment:
this was for the case of showing external annotations, which often do not have notes.